tests/scripts/ping_reviewers.py

```diff
@@ -128,17 +147,19 @@ def check_pr(pr, wait_time, now):
 
     if time_since_last_action > wait_time:
         print(
-            "    Pinging reviewers",
+            "  Pinging reviewers",
             reviewers,
             "on",
             pr["url"],
             "since it has been",
             time_since_last_action,
-            "since anything happened on that PR",
+            f"since anything happened on that PR (last action: {last_action[1]})",
         )
         return reviewers
     else:
-        print(f"Not pinging PR {pr['number']}")
+        print(
+            f"  Not pinging PR {pr['number']} since it has been only {time_since_last_action} since the last action: {last_action[1]}"
```

Comment:
could you add units to time print?

Reply:
it’s a formatted time so something like hh:mm:ss (so there’s not really a unit to add)